src/components/FVApps/FVApps.js

```diff
@@ -1,34 +1,91 @@
+import { SMALL } from 'common/constants'
+import SiteLogo from 'components/SiteLogo'
+import getIcon from 'common/utils/getIcon'
+import GlobalConfiguration from 'src/GlobalConfiguration'
 
-function FVApps() {
+function FVApps({ hasApp }) {
   const headerStyle = 'text-xl font-bold mb-1 mt-4'
   const paraStyle = 'mb-2'
+
   return (
     <section
       className="pt-2 md:pt-4 lg:pt-8 bg-white"
       data-testid="FirstVoicesApps"
     >
-      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+      <div className="max-w-6xl mx-auto px-4 sm:px-6 lg:px-8">
         <SectionTitle.Presentation
           title="FIRSTVOICES APPS"
           accentColor="primary"
         />
-        <div className="max-w-4xl mx-auto text-fv-charcoal space-y-4 py-8">
+        <div className="max-w-5xl mx-auto text-fv-charcoal space-y-4 py-8">
           <h2 className={headerStyle}>FirstVoices Language Apps</h2>
           <p className={paraStyle}>
             The wealth of language data uploaded by Indigenous communities onto
             their FirstVoices language sites can also be accessed through mobile
-            apps. The app pulls content directly from entries on FirstVoices and
-            functions with offline capabilities.
+            apps. The app pulls content directly from FirstVoices and is
+            compatible with most phones, tablets, and laptops.
           </p>
           <p className={paraStyle}>
-            Note: The classic FirstVoices language apps are no longer being
-            updated and may not be installable on all devices. Watch out for the
-            new and improved FirstVoices language apps with automatic syncing
-            capabilities, coming soon!
+            The new apps can be installed directly from the web
+          </p>
+          {hasApp && (
+            <div className="grid grid-cols-3 gap-6 pt-8">
+              {hasApp?.map(({ id, title, slug, logo }) => (
+                <div
+                  className="rounded-lg bg-white p-6 drop-shadow-md flex flex-col items-center"
+                  key={id}
+                >
+                  <div className="h-16 w-16 md:w-24 md:h-24">
+                    <SiteLogo.Presentation
+                      size={SMALL}
+                      logo={logo || null}
+                      additionalStyling="ring-1 ring-gray-200"
+                    />
+                  </div>
+                  <h2 className="font-bold p-4">{title}</h2>
+                  <div className="pb-6 w-2/3 flex justify-between">
+                    {getIcon('Mobile', 'w-6 h-6 fill-current mr-3 inline-flex')}
+                    {getIcon('Tablet', 'w-6 h-6 fill-current mr-3 inline-flex')}
+                    {getIcon('PC', 'w-6 h-6 fill-current mr-3 inline-flex')}
+                  </div>
+                  <a
+                    href={
+                      GlobalConfiguration.ENV_APP === 'production'
+                        ? `https://${slug}.firstvoicesapp.com`
```

Comment:
Since we are already using a variable to hold configuration here, why not put the entire base url for the app into it? We theoretically have a goal of other people being able to reuse this code, and every time we hardcode FirstVoices-specific configuration or branding it makes that harder

Reply:
it's because we need slug to be mapped in the jsx